nessie/src/main/java/org/apache/iceberg/nessie/NessieCatalog.java

```diff
@@ -262,11 +262,10 @@ public List<Namespace> listNamespaces(Namespace namespace) throws NoSuchNamespac
   }
 
   /**
-   * Load the given namespace but return an empty map because namespace properties are currently not
-   * supported.
+   * Load the given namespace and return its properties.
    *
    * @param namespace a namespace. {@link Namespace}
-   * @return an empty map
+   * @return properties map
```

Comment:
maybe `a string map of properties for the given namespace`

Reply:
SGTM. Updated